catapult-sdk/test/plugins/receipts_spec.js

```diff
@@ -59,7 +59,7 @@ describe('receipts plugin', () => {
 			// - mosaicResolutionStatement
 			// - transactionStatement
 			['addressResolution', 'mosaicResolution', 'transaction'].forEach(statementType => {
-				expect(Object.keys(modelSchema[`${statementType}Statement`]).length).to.equal(2);
+				expect(Object.keys(modelSchema[`${statementType}Statement`]).length).to.equal(3);
 				expect(modelSchema[`${statementType}Statement`]).to.contain.all.keys(['statement']);
```

Comment:
you might want to update this check

Reply:
Ok, I see, thanks. Would we want to add timestamp to `addressResolutionStaments` and `mosaicResolutionStatements`? 

To these endpoints?

http://ngl-dual-101.testnet.symboldev.network:3000/statements/resolutions/address
http://ngl-dual-101.testnet.symboldev.network:3000/statements/resolutions/mosaic

Or separate transactionStements from the other 2 is fine?

Reply:
either is fine

Reply:
https://github.com/symbol/symbol-openapi/issues/279 for the open api change